### DISCUSSION TITLE
Agile workflow

### DISCUSSION THREAD

**Discussion by paulpestov** [OCR-D/core#711] [General]
The idea is to continue development of ocr-d core in a more focused way. For this we could evaluate how agile methods for software development can be benefitial in our situation. Keeping a backlog of small self containing chunks of work is the essence of those methods. 

With the example of the "coordinate" board we would be interested in opinions about the advantages and disadvantages of several backlog solutions. Possible questions:
- Who should work with which board? (Should active developers have a clear board with tasks?)
- Keep a common board or split up by categories?
- Content management tasks separately?

**Comment by kba** [comment]
We could have a "master" board with all the issues from all the repositories for the big picture and dedicated boards for individual repos or task-focussed boards (e.g. everything related to the API changes that require adaption of processors).

**Reply by bertsky** [reply]
I really like persistent master plus transient task-focussed boards (perhaps tied to GH projects?), but not personalized (and ideally open to all collaborators for writing and everyone for reading)
